<launch>
    <group ns="rtabmap">
        <node pkg="nodelet" type="nodelet" name="rtabmap_manager" args="manager" output="screen" />

        <node pkg="nodelet" type="nodelet" name="obstacle" args="load rtabmap_ros/obstacles_detection rtabmap_manager" output="screen">
            <remap from="cloud" to="/stereo_camera/voxel_grid/output" />
        </node>
    </group>
</launch>